### PR TITLE
Layout for Quick preview

### PR DESCRIPTION
**Pull request by SMahil**
*** **WIP** ***

This PR includes 
- Layout for Quick Preview
- Icon at list view
- Fix CSS for quick preview panel(Inside content including WI Links and Comments)

**This doesnot include name change as for now. We have dependency on Full Page Detail View. Will do the file name change at that time

### CODE REVIEW

File: src/app/components/work-item-detail/work-item-detail.component.html [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
+  <div class="width-100" *ngIf="workItem" id="workItemDetail_Wrapper">
     <form *ngIf="active"
           #almWorkItemDetailForm="ngForm"
           class="form-horizontal"
           id="wi-detail-form">
       <fieldset>
-        <div class="detail-panel-header">
-          <div class="form-group col-xs-12 col-sm-12 col-md-12 mobile-bar">
-            <div class="dropdown-kebab-pf detail-type-dropdown col-sm-8">
+        <div class="f8-quick-preview__header">
```
Comment (sanbornsen) [inline src/app/components/work-item-detail/work-item-detail.component.html]: Why is '_' used instead of '-' that also two of them ?

Comment (SMahil) [inline src/app/components/work-item-detail/work-item-detail.component.html]: for Element we are using two underscores. As one underscore we are using for id naming converntion

File: src/app/components/work-item-detail/work-item-detail.component.html [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
+          <span
+            class="pficon pficon-close pull-right f8-quick-preview--close"
```
Comment (SMahil) [inline src/app/components/work-item-detail/work-item-detail.component.html]: close is modifier. As per BEM Technology We are naming it like that

File: src/app/components/typeahead-dropdown/typeahead-dropdown.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
-<div class="col-sm-10 typeahead-dropdown combobox-container">
+<div class="width-100 pull-left typeahead-dropdown combobox-container">
     <span *ngIf="!searchValue" (click)="open()" class="pointer details-dropdown" [ngClass]="getInitialValue().cssLabelClass?getInitialValue().cssLabelClass:''">{{ getInitialValue().value }}</span>
-    <div *ngIf="searchValue" class="dropdown-search col-md-6 padding-0">
+    <div *ngIf="searchValue" class="dropdown-search col-md-6 padding-0" style="padding:0">
```
Comment (vikram-raj) [inline src/app/components/typeahead-dropdown/typeahead-dropdown.component.html]: Why did you add inline CSS here?

Comment (SMahil) [inline src/app/components/typeahead-dropdown/typeahead-dropdown.component.html]: I forgot to remove it.. will update

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2105-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2105-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2105-7 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2105-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2105-8 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2105-fabric8-planner.badger.fabric8.io
